### DISCUSSION TITLE
learn/articles/

### DISCUSSION THREAD

**Discussion by giscus** [CommandPost/FCPCafe#97] [Website Discussions]
# Articles

This section of the site is dedicated to linking to great external articles, as well as providing a space to write FCP Cafe specific articles.

https://fcp.cafe/articles/

**Comment by LifeSMyth** [comment]
Are these FCP.co articles archived anywhere else. [like how about here?]
All links redirect to the Industrial Revolution web site.

**Reply by latenitefilms** [reply]
I’ll eventually update the links to point to the Internet Archive. There’s no easy way to reproduce them here due to copyright, etc.
